docs/book/src/developer/release.md

```diff
@@ -25,30 +25,35 @@
 > Note: In the above instructions, `v0.2.0-alpha.3` is the version/tag is being released
 
 ## GA Releases
-- Create a tag and push
+- Review if all issues linked to the release version are either completed or moved to the "Next" release.
+- Create a release branch from main.
+- Clone the repository and create a tag (release tag) and push to origin. Ensure that the GPG keys are set.
     ```shell
     git clone git@github.com:kubernetes-sigs/cluster-api-provider-ibmcloud.git
     git tag -s -m "v0.1.0" v0.1.0
     git push origin v0.1.0
     ```
-- Wait for the google cloud build to be finished
-- [Prepare release notes](#prepare-release-notes)
-- Create a draft release with release notes for the tag
-- Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter):
+- Wait for the Google Cloudbuild to finish, which is triggered once the tag is created.
+  - The status of the build jobs can be tracked from : https://prow.k8s.io/?job=post-cluster-api-provider-ibmcloud-push-images
+  - The built images are available here: https://console.cloud.google.com/gcr/images/k8s-staging-capi-ibmcloud/
```

Comment:
The links do not appear as a hyperlink in the doc - https://deploy-preview-2223.cluster-api-ibmcloud.sigs.k8s.io/developer/release

Reply:
I see that's the case with other links too. For eg: 
https://deploy-preview-2223.cluster-api-ibmcloud.sigs.k8s.io/topics/vpc/uploading-an-image.html?highlight=https#add-vm-image-to-vpc

Should we consider taking it up in a separate issue?

Reply:
sure, we can fix the hyperlinks as a separate issue.